hack/install-operator-sdk.sh

```diff
@@ -9,7 +9,7 @@
 set -e
 
 DEST="${1:-bin/operator-sdk}"
-SDK_VERSION="${SDK_VERSION:-1.4.2}"
+SDK_VERSION="${SDK_VERSION:-1.9.0}"
```

Comment:
Seems there is a version number mismatch (`1.9.0` vs `1.8.0+git` in the make bundle update)

Reply:
i think this might be some sort of bug 🤔 of the `operator-sdk`, this is the bundle Dockerfile generated by `1.9.0` `operator-sdk`..

Reply:
I think you are right - do the operator-sdk folks know?